pages/_admin/index.tsx

```diff
@@ -42,69 +49,69 @@ function Story({ id, title, content, name, postal, ...rest}) {
 }
 
 export default function _Admin({ stories }) {
-  const [ session ] = useSession()
+  const [session] = useSession()
 
-  return <>
+  return (
+    <>
       <Nav session={session} />
-      {session && <>
-        <Container>
-          <Stack spacing={8}>
-            {stories.map((story) => (
-              <Story
-                {...story}
-             />
-            ))}
-
-          </Stack>
-        </Container>
-      </>}
-  </>
+      {session && (
+        <>
+          <Container>
+            <Stack spacing={8}>
+              {stories.map((story) => (
+                <Story key={story.id} {...story} />
+              ))}
+            </Stack>
+          </Container>
+        </>
+      )}
+    </>
+  )
```

Comment:
@silent1mezzo Just formatting tidy-up - you should install prettier and eslint plugins :)

Reply:
Hmm, I have. Will figure this out